src/Operation/FindAndModify.php

```diff
@@ -239,13 +239,17 @@ public function execute(Server $server)
         return $result->value;
     }
 
+    public function getCommandDocument()
+    {
+        return $this->createCommandDocument();
+    }
+
     /**
-     * Create the findAndModify command.
+     * Create the findAndModify command document.
      *
-     * @param Server $server
-     * @return Command
+     * @return array
      */
-    private function createCommand(Server $server)
+    private function createCommandDocument()
```

Comment:
This definitely needs a Server passed in, both for Explainable and creating the actual command for `execute()`.